pkg/consoleuser/consoleuser_darwin.go

```diff
@@ -0,0 +1,45 @@
+//go:build darwin
+// +build darwin
+
+package consoleuser
```

Comment:
At least for now, I'd drop these in `ee`. They're only used by the ee functions. But if we ever make a table, I'll say we should move them